xpdacq/tools.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# xpdacq            by Billinge Group
+#                   Simon J. L. Billinge sb2896@columbia.edu
+#                   (c) 2016 trustees of Columbia University in the City of
+#                        New York.
+#                   All rights reserved
+#
+# File coded by:    Timothy Liu
+#
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
+#
+##############################################################################
+def clean_dict(input_dict, target_chr, replace_chr):
```

Comment:
Would it be possible to name this something a bit more descriptive? Like `replace_dict_keys`? Cleaning a dictionary can mean a lot of things.